src/porepy/numerics/interface_laws/elliptic_interface_laws.py

```diff
@@ -715,12 +716,9 @@ def assemble_matrix_rhs(
         self.discr_master.enforce_neumann_int_bound(
             g_master, data_edge, matrix, False, master_ind
         )
-
-        # Consider this terms only if the grids are of the same dimension
-        if g_master.dim == g_slave.dim:
-            self.discr_slave.enforce_neumann_int_bound(
-                g_slave, data_edge, matrix, True, slave_ind
-            )
```

Comment:
The finite volume schemes does not use the enforce_neumann_int_bound, but my guess is that the first is correct. Otherwise are you will try to enforce Neumann conditions on the cells of the lower dim grid?

Reply:
so I should reverse the change to the original one?

Reply:
I think so.

Reply:
I think so.